examples/Lorenz/Lorenz_example_ukp.jl

```diff
@@ -70,26 +70,25 @@ println(params_true)
 ###
 ###  Define the parameter priors
 ###
-# Lognormal prior or normal prior?
-log_normal = false # THIS ISN't CURRENTLY IMPLEMENTED
-
-function logmean_and_logstd(μ, σ)
-    σ_log = sqrt(log(1.0 + σ^2 / μ^2))
-    μ_log = log(μ / (sqrt(1.0 + σ^2 / μ^2)))
-    return μ_log, σ_log
-end
-#logmean_F, logstd_F = logmean_and_logstd(F_true, 5)
-#logmean_A, logstd_A = logmean_and_logstd(A_true, 0.2*A_true)
-
 if dynamics == 2
-    prior_mean = [F_true + 1.0; A_true + 0.5]
-    # prior_stds = [2.0, 0.5*A_true]
-    prior_cov = Array(Diagonal((0.1 * prior_mean) .^ 2))
+    prior_means = [F_true + 1.0, A_true + 0.5]
+    prior_stds = [2.0, 0.5 * A_true]
+    prior_F = constrained_gaussian(param_names[1], prior_means[1], prior_stds[1], -Inf, Inf)
+    prior_A = constrained_gaussian(param_names[2], prior_means[2], prior_stds[2], -Inf, Inf)
+    priors = combine_distributions([prior_F, prior_A])
 else
-    prior_mean = [F_true;]
-    prior_cov = ones(Float64, 1, 1)
+    priors = constrained_gaussian("F", F_true, 1.0, -Inf, Inf)
 end
 
+# if dynamics == 2
+#     prior_mean = [F_true + 1.0; A_true + 0.5]
+#     # prior_stds = [2.0, 0.5*A_true]
+#     prior_cov = Array(Diagonal((0.1 * prior_mean) .^ 2))
+# else
+#     prior_mean = [F_true;]
+#     prior_cov = ones(Float64, 1, 1)
+# end
```

Comment:
delete?

Reply:
Done, and removed a bunch of other unnecessary comments